packages/vuetify/src/components/VBtn/__tests__/VBtn.spec.ts

```diff
@@ -1,3 +1,5 @@
+declare const expect: jest.Expect
```

Comment:
Would we have to add this to every non-cypress test file?

Reply:
Yes, if you want the Jest types on the expect matchers. The alternate solution is to pull out the Cypress tests into their own directory. Alternatively you can `import { expect} from 'jest'`.

Reply:
I think this works for now.